File: Business Up/Business Up/Entidades/Servicio.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Business_Up.Entidades
{
    [Serializable]
    public class Servicio
    {
        private string nombre;
        private double valor;
        private string codigoBarra;

        public Servicio()
        {

        }

        public Servicio(string nombre, double valor, string codigoBarra)
        {
            this.nombre = nombre;
            this.valor = valor;
            this.codigoBarra = codigoBarra;
        }

        public string Nombre { get => nombre; set => nombre = value; }
        public double Valor { get => valor; set => valor = value; }
        public string CodigoBarra { get => codigoBarra; set => codigoBarra = value; }
    }
}

File: BusinessUpAndroid/app/src/main/java/com/businessup/data/model/Servicio.kt
package com.businessup.data.model

import android.os.Parcelable
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.parcelize.Parcelize

@Parcelize
@Entity(tableName = "servicios")
data class Servicio(
    @PrimaryKey(autoGenerate = true)
    val id: Long = 0,
    val nombre: String,
    val valor: Double,
    val codigoBarra: String = ""
) : Parcelable {

    val datoValor: String
        get() = "$${"%.2f".format(valor)}"
}
